app/views/certificates/show.pdf.erb

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="http://fonts.googleapis.com/css?family=Arvo|Alex+Brush|Open+Sans:300|La+Belle+Aurore" rel="stylesheet" type="text/css">
+    <style>
+      html, body {
+        height: 100%;
+      }
+      html {
+        border: 10px solid #adadad;
+      }
+      body {
+        border: 1px solid #adadad;
+        padding: 50px;
+
+        background-image: url("http://www.cs.helsinki.fi/u/josalmi/watermark17.png"), url("http://www.cs.helsinki.fi/u/josalmi/watermark17.png");
+        background-repeat: no-repeat;
+        background-position: top center,right bottom;
+        background-position-y: -150%, 170%;
+        background-position-x: 70%, 130%;
+      }
+
+      #ribbon {
+        position: absolute;
+        top: 1px;
+        text-align: center;
+      }
+
+      #ribbon p {
+        color: #3dbba6;
+        font-size: 2em;
+      }
+
+      #hy-logo {
+        position: absolute;
+        margin-left: 10px;
+        top: 100px;
+      }
+      #logo {
+        position: absolute;
+        margin-left: 10px;
+        bottom: 75px;
+      }
+
+      main {
+        position: absolute;
+        left: 450px;
+        top: 85px;
+        width: 500px;
+      }
+      .date {
+        font-size: 0.8em;
+      }
+      body {
+      font-family: 'Open Sans';
+      font-weight: 300;
+      color: #333333;
+      }
+      .signature {
+        font-family: 'La Belle Aurore', sans;
+        font-size: 1.9em;
+        border-bottom: 1px dotted #acacac;
+        width: 250px;
+        line-height: 12px;
+        text-align: center;
+        margin-bottom: 5px;
+        margin-top: 50px;
+      }
+      .name, .title {
+         font-size: 0.8em;
+        margin-top: 0;
+        margin-bottom: 0;
+      }
+      #verify {
+        font-size: 0.8em;
+        position: absolute;
+        right: 30px;
+        padding: 0;
+        margin: 0;
+        bottom: 75px;
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <img width="280" id="logo" src="http://cs.helsinki.fi/u/josalmi/cert/logo.png">
+    <img width="280" id="hy-logo" src="http://cs.helsinki.fi/u/josalmi/cert/hy-logo.png">
+    <main>
+      <p class="date"><%= Time.now.strftime("%^B %d, %Y") %></p>
+      <h1>Joni Salmi</h1>
+      <p>has successfully completed the</p>
+      <h2><%= @course.formal_name %></h2>
+      <p>course offered by the University of Helsinki Department of Computer Science. Course consisted of <%= @course.exercise_groups.length %> weeks and <%= @visible_exercises.size %> exercises. Total score of the user was <%= "#{@points} / #{@total_available}" %>.</p>
+      <p class="signature">Matti Luukkainen</p>
+      <p class="name">Matti Luukkainen</p>
+      <p class="title">University lecturer</p>
```

Comment:
Maybe these should not be hard coded; e.g. this wouldn't work for the Aalto C course...